host_pool_plugin/tasks.py

```diff
@@ -0,0 +1,65 @@
+# #######
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+import httplib
+import os
+import requests
+
+from cloudify import ctx
+from cloudify.exceptions import NonRecoverableError
+from cloudify.decorators import operation
+
+
+@operation
+def acquire(service_url, **kwargs):
+    ctx.logger.info("Acquire host")
+    response = requests.post('{0}/hosts'.format(service_url))
+    host = response.json()
+    ctx.logger.info("Response received: {0}".format(str(host)))
+    if response.status_code == httplib.CREATED:
+        ctx.instance.runtime_properties['ip'] = host['host']
```

Comment:
i would add a log here in the lines of:

    ctx.logger.info('Acquired host: {0}'.format(host['host']))